deployment/v3/external/iam/values.yaml

```diff
@@ -9,7 +9,8 @@ auth:
 
 extraEnvVars:
   - name: KEYCLOAK_EXTRA_ARGS
-    value: "-Dkeycloak.profile.feature.upload_scripts=enabled"
+    value: "-Dkeycloak.profile.feature.upload_scripts=enabled -Dkeycloak.profile.feature.token_exchange=enabled
+            -Dkeycloak.profile.feature.admin_fine_grained_authz=enabled"
```

Comment:
It should be in a single line

    value: "-Dkeycloak.profile.feature.upload_scripts=enabled -Dkeycloak.profile.feature.token_exchange=enabled
 -Dkeycloak.profile.feature.admin_fine_grained_authz=enabled"